data/jubeat.tsv

```diff
@@ -1108,12 +1120,171 @@
 80001008	カラルの月 [ 2 ]	猫叉劇団
 80001009	君の知らない物語 [ 2 ]	
 80001010	白い雪のプリンセスは [ 2 ]	のぼる↑
-80001011	月に叢雲花に風 [ 2 ]	幽閉サテライト feat. senya
+80001011	月に叢雲花に風 [ 2 ]	幽閉サテライト?feat.?senya
 80001013	Scars of FAUNA [ 2 ]	猫叉Master
 80001014	chaplet [ 2 ]	DJ TOTTO
 80001015	アガット [ 2 ]	Bazole
-90000002	インドがパッカーン！！煩悩マハラジャドリーム	ARM × かたほとり ft. 柊木りお
-90000003	ロミとロボの宇宙飛行	onoken
+90000001	Samba Ramba	Y&Co. feat. Kanae Asaba
+90000002	インドがパッカーン！！煩悩マハラジャドリーム	ARM × かたほとり ft. 柊木りお
+90000003	ロミとロボの宇宙旅行	onoken
 90000004	Sahara	Hommarju
 90000005	Phantasmagoria	Maozon
 90000006	EMOTiON TRiPPER	sky_delta
+90000007	恋 No リセットゲーム!!!!!	"DJ Genki feat. ななひら,yukacco,藍月なくる"
```

Comment:
Woo! I *hate* transcribing this stuff too.

Reply:
Thank you for this list though. It was SUPER helpful for a different project i was working on that involved converting all the charts in the game to a format that I could play in a simulator